package.json

```diff
@@ -319,7 +320,48 @@
                         ],
                         "type": "object"
                     }
-                }
+                },
+                "configurationSnippets": [
+                    {
+                        "label": "%AWS.configuration.description.awssam.debug.snippets.lambdaCode.label%",
+                        "description": "%AWS.configuration.description.awssam.debug.snippets.lambdaCode.description%",
+                        "body": {
+                            "type": "aws-sam",
+                            "request": "direct-invoke",
+                            "name": "${3:Invoke Lambda}",
```

Comment:
What value does the ${3:...} provide? The gif doesn't make it clear.

Reply:
It is the tab order to fill in the blanks after the snippet is created

Reply:
^^ this.